ui/src/utils/helpers.js

```diff
@@ -1,5 +1,7 @@
 import { format, formatDuration, intervalToDuration } from "date-fns";
 import _ from "lodash";
+import { isEmpty as _isEmpty } from 'lodash';
```

Comment:
I recommend using `import _isEmpty from 'lodash/isEmpty'`.
Btw the above line imported `lodash`, so you need to use `_.isEmpty` only, don't need to re-import.

Reply:
Thanks for the feedback. I have fixed this by removing my duplicate import and using `_.isEmpty`